.github/workflows/build-fedora.yml

```diff
@@ -0,0 +1,48 @@
+name: Build Fedora
+
+on:
+  push:
+    branches: [ "qml" ]
+  pull_request:
+    branches: [ "qml" ]
+
+
+env:
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    name: Fedora ${{ matrix.fedora_version }}
```

Comment:
Fedora isn't an OS I'm looking to provide explicit support for, so this is probably best suited to a fork.